my_portfolio.html

```diff
@@ -1,13 +1,63 @@
-﻿<!DOCTYPE html>
-
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-    <h1>Hello world</h1>
+    <div id="header-img"></div>
+    <div id="header-img-desktop"></div>
```

Comment:
Great work! Regarding the previous reviewer's request, there is an optional improvement to consider:

- [ ] [Optional] You can add the background to the `<header>` tag for consistency. As suggested before, please remember to remove the two unnecessary `<div>` tags. Finally, apply the background styling to the `<header>`.